tests/cmake/X3D_CREATE_TEST.cmake

```diff
@@ -1,7 +1,6 @@
 macro(CreateMPITest test_dir
 		 case
 		 input_file)
-  message(STATUS "Add Test ${case}")
```

Comment:
I think it can be useful to have these messages to confirm what tests are being generated. Perhaps we could make them more useful by `Add <MPI | MPI + Python> Test ${case}`?